src/RollbarLogHandler.php

```diff
@@ -73,55 +74,11 @@ public function log($level, $message, array $context = [])
             return;
         }
 
-        $context = $this->addContext($context);
```

Comment:
@olafnorge we should add person info, it's possible with $logger->configure() https://github.com/jenssegers/laravel-rollbar/pull/84/files#diff-e31215fa03abb6aa2a7a094e2286af78R134